config/default/kustomization.yaml

```diff
@@ -6,7 +6,7 @@ namespace: argocd
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: apr
+namePrefix: progressive-sync-
```

Comment:
the extra `-` in the end isn't gonna cause any issue i presume?

Reply:
No I think the old one is wrong but since we don't use kustomize we never realized that. This is a prefix so it makes sense to have the `-`